.github/workflows/documentation.yml

```diff
@@ -17,46 +16,22 @@ jobs:
     runs-on: ubuntu-22.04
 
     steps:
-      - uses: actions/checkout@v4
-      - uses: actions/setup-python@v4
-        with:
-          python-version: '3.10'
-
-      - id: get-version
-        run: echo "ls_version=$(git rev-parse --short HEAD)" >> $GITHUB_OUTPUT
-
-      - id: setup-ssh
-        run: |-
-          mkdir -m 700 -p ~/.ssh
-          echo "$SSH_KEY" > ~/.ssh/id_ed25519
-          echo "$HOST_KEY" > ~/.ssh/known_hosts
-          chmod 600 ~/.ssh/*
-          echo "have_ssh_key=yes" >> $GITHUB_OUTPUT
-        env:
-          SSH_KEY: ${{secrets.WEB1_DOCS_SECRET}}
-          HOST_KEY: ${{vars.WEB1_HOSTKEY}}
-        if: ${{ (github.ref_name == 'main' || github.ref_name == 'docs') && env.SSH_KEY != '' }}
-
-      - run: python3 -m venv .venv
-      - run: .venv/bin/pip3 install -r docs/requirements.txt
-      - run: .venv/bin/mkdocs build
-      - run: mv site lightningstream-html-docs
-      - run: tar cf lightningstream-html-docs.tar lightningstream-html-docs
-      
-      - uses: actions/upload-artifact@v4
+      - uses: actions/checkout@v6
+      - name: Check for semver release tag # note the $ in the regex which excludes pre-releases
+        id: release_check
+        run: |
+          if [[ ${{ github.ref }} =~ ^refs/tags/v[0-9]+\.[0-9]+\.[0-9]+$ ]]; then
+            echo "release=YES" >> $GITHUB_OUTPUT
+            echo "version_string=${{ github.ref }}" | sed -r 's/(=)v/\1/' >> $GITHUB_OUTPUT
```

Comment:
You've just checked for github.ref to be refs/tags/vX.Y.Z, should this be `sed -r 's%(=)refs/tags/v%\1%'`?